docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md

```diff
@@ -16,95 +16,129 @@ If you are still testing or prototyping your script, it is recommended that you
-you can proceed with this step.
-In this step, you will be creating a module which will deploy your Glue job (and crawler) to the Data Platform Project.
+Once the Pull Request to add or update your script has been approved, and you have confirmed that your script is in the Data Platform Project,
+you can proceed to follow the steps in this section.
 
-- Open the [Data Platform Project][data-platform-project] in GitHub
-  If you don't have the correct permissions, you'll get a '404' error (see [prerequisites][prerequisites]).
-- Navigate to the main `terraform` directory. Open `23-aws-glue-jobs-<YOUR-DEPARTMENT-NAME>.tf`
-  - _if this file does not exist, create a new terraform file for your department by clicking `Add file` then `Create new file`.
-    The name of the new file should be:_
-    ```
-    23-aws-glue-jobs-<YOUR-DEPARTMENT-NAME>.tf
-    ```
-    _Refer to the [example](#example-module-block) below to get started._
-- Click `edit` or the pencil icon and copy and paste an existing module block.
-- When updating the module:
-  - Do not change any spacing before the `=`
-  - Do not remove the `"quotation marks"` around any values which use them
-  - When adding a variable that does not already exist, paste the variable name and then intended value after the `=`
-    and ensure that all equal signs (`=`) line up as shown in the example
+In this section, you will be creating a module (or updating an existing one if you need to make changes to an existing Glue job) which will deploy your Glue job (and crawler) to the Data Platform Project.
+
+1. Open the [terraform directory][terraform-directory] in the Data Platform Project in GitHub.
+
+   - If you don't have the correct permissions, you'll get a '404' error (see [prerequisites][prerequisites]).
+
+1. Open `25-aws-glue-jobs-<YOUR-DEPARTMENT-NAME>.tf`. e.g. `25-aws-glue-job-parking.tf`
+
+   - _If this file does not exist, create a new terraform file for your department by clicking `Add file` then `Create new file`.
+     The name of the new file should be:_
+
+   ```
+   25-aws-glue-jobs-<YOUR-DEPARTMENT-NAME>.tf
+   ```
+
+   - _Refer to the [example](#example-module-block) below to get started._
+
+1. Click `edit` or the **pencil icon** (:pencil2:) and copy and paste an existing module block.
+   An example of what a module block looks like can be seen [here][project-module-example].
 
-Update the `module` name using the name convention `<job_name>_<department_name>`, for example "liberator_pcn_denormalisation_housing_repairs".
-The module name must be all lowercase with words separated by underscores. Ensure it's unique to all other module names in this file.
+1. Update the `module` name using the name convention `<job_name>_<department_name>`, for example: `"liberator_pcn_denormalisation_housing_repairs"`.
+
+   - The `job_name` must be the same as the script name you specified in the previous section (without the `.py` extension).
```

Comment:
This step is talking about the module name, maybe move this line to be with the section where you specify the job name.

Reply:
I've made it clearer that it refers to the `<job_name>` in module name: `<job_name>_<department_name>`

Reply:
And also added that line to the `job_name` variable